user_management/src/user/views.py

```diff
@@ -113,15 +113,16 @@ def post(self, request):
             validation_errors = SignInView.signin_infos_validation(json_request)
             if not validation_errors:
                 user = User.objects.filter(username=json_request['username']).first()
-                refresh_token = JWTManager('refresh').generate_token(user.id)
+                success, refresh_token, errors = JWTManager('refresh').generate_token(user.id)
+                if success is False:
+                    return JsonResponse(data={'errors': errors}, status=400)
                 return JsonResponse(data={'refresh_token': refresh_token}, status=200)
-            else:
-                return JsonResponse(data={'errors': validation_errors}, status=400)
+            return JsonResponse(data={'errors': validation_errors}, status=400)
```

Comment:
You can do:
```python
if validation_errors:
    return JsonResponse(...)
user = ...
```
Instead of:
```python
if not validation_errors:
    user = ...
return JsonResponse(...)
```